patches/minecraft/net/minecraft/world/entity/EntityType.java.patch

```diff
@@ -119,7 +119,8 @@
 +      }
 +
        public EntityType<T> m_20712_(String p_20713_) {
-          if (this.f_20688_) {
+-         if (this.f_20688_) {
++         if (this.f_20688_ && false) {
```

Comment:
Shouldn't the false condition to skip this be either added at the start of this if statement for readability. (potentially adding it as a single line if above to reduce patch size slightly)